.github/workflows/ci.yml

```diff
@@ -81,3 +81,39 @@ jobs:
       - name: Upload Coverage Reports
         if: success()
         run: Scripts/upload-coverage-reports.sh ${{ matrix.platforms }}
+  spm-13:
+    name: Build Xcode 13
+    runs-on: macOS-12
+    strategy:
+      matrix:
+        platforms: [
+          'iOS_15,tvOS_15,watchOS_8',
+          'macOS_12',
+        ]
+      fail-fast: false
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Bundle Install
+        run: bundle install
+      - name: Select Xcode Version
+        run: sudo xcode-select --switch /Applications/Xcode_13.4.1.app/Contents/Developer
```

Comment:
JFYI `xcversion` comes installed in these machines, which would make selecting an Xcode much easier than this https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#xcode-support-tools

Reply:
Nice! de95961